app-lifelike/src/main/java/com/example/lifelike/composable/LoggedOut.kt

```diff
@@ -1,11 +1,7 @@
 package com.example.lifelike.composable
 
 import androidx.compose.Composable
-import com.example.lifelike.composable.loggedout.RegConfirmSmsCode
-import com.example.lifelike.composable.loggedout.RegFinal
-import com.example.lifelike.composable.loggedout.RegUserName
-import com.example.lifelike.composable.loggedout.RegUserPhone
-import com.example.lifelike.composable.loggedout.Splash
+import com.example.lifelike.composable.loggedout.*
```

Comment:
Star imports are considered a bad practice, because you can accidentally import something extra from a package.
Can you please revert it?

You can also update studio to avoid these changes in `Preferences | Editor | Code Style | Kotlin | Imports` and select `Use single name import` everywhere :)